huggingface/base/inference/docker/Dockerfile.cpu

```
@@ -0,0 +1,119 @@
+FROM ubuntu:18.04
+
+LABEL maintainer="Amazon AI"
+LABEL dlc_major_version="1"
+
+# Specify accept-bind-to-port LABEL for inference pipelines to use SAGEMAKER_BIND_TO_PORT
+# https://docs.aws.amazon.com/sagemaker/latest/dg/inference-pipeline-real-time.html
+LABEL com.amazonaws.sagemaker.capabilities.accept-bind-to-port=true
+# Specify multi-models LABEL to indicate container is capable of loading and serving multiple models concurrently
+# https://docs.aws.amazon.com/sagemaker/latest/dg/build-multi-model-build-container.html
+LABEL com.amazonaws.sagemaker.capabilities.multi-models=true
+
+ARG MMS_VERSION=1.1.2
+ARG PYTHON=python3
+ARG PYTHON_VERSION=3.7.10
```

Why have you chosen `PYTHON_VERSION=3.7.10` when all other inference DLC use `3.6.13` and the aws optimized PyTorch only supports `py36`? 
```suggestion
ARG PYTHON_VERSION=3.6.13
```

---

Yup. I need to revert this.